_config.yml

```diff
@@ -84,16 +84,13 @@ collections:
 
 # Sets breadcrumbs per path
 defaults:
+
   - scope:
       type: "disability-benefits"
     values:
       layout: "page-breadcrumbs"
       body_class: "page-disability"
   - scope:
-      path: "disability-benefits/conditions/exposure-to-hazardous-materials"
-    values:
-      breadcrumb_3: "Exposure to Hazardous Materials"
-  - scope:
       path: "disability-benefits/apply-for-benefits/"
```

Comment:
Why did we remove this here? Not sure if this was intentional - or just an instance of V2 being out of sync with master